build.gradle

```diff
@@ -87,7 +87,8 @@ subprojects {
         bndVersion = '2.4.1'
     }
 
-    if (project.name == 'alfresco-dynamic-extensions-repo' || project.parent.name == 'alfresco-dynamic-extensions-repo') {
+    if (project.name == 'alfresco-dynamic-extensions-repo' || project.parent.name == 'alfresco-dynamic-extensions-repo'
+            || project.name == 'gradle-plugin') {
```

Comment:
I think we still to want to execute the `tasks.withType(JavaCompile)` part below?

Reply:
Good point